src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,78 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int LOG_MIN_LEN = 1;
```

Comment:
Why all your constant public? Don’t use abbreviations for words. you can do this only for commonly accepted abbreviations like ```min```, ```max```. But what does it mean ```LEN```?
```suggestion
    private static final int LOGIN_MIN_LENGTH = 1;
```